concent_api/core/message_handlers.py

```diff
@@ -822,14 +822,20 @@ def store_subtask(
     provider  = Client.objects.get_or_create_full_clean(provider_public_key)
     requestor = Client.objects.get_or_create_full_clean(requestor_public_key)
 
+    computation_deadline = task_to_compute.compute_task_def['deadline']
```

Comment:
Blank lines above and below not needed. Beside that, if those are used only once they can be passed directly to logging function.

Reply:
Those are used twice.